introduction.md

```diff
@@ -0,0 +1 @@
+Hello, my Name is Kevin, and I am new to the Dojo.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.